ops/terraform/modules/resources/asg/main.tf

```diff
@@ -94,7 +94,7 @@ resource "aws_launch_template" "main" {
   }
 
   block_device_mappings {
-    device_name = "/dev/sda1"
+    device_name = "/dev/xvda"
```

Comment:
Update root volume nomenclature to work with AL2.  Strangely this was not required for the ETL build (EC2 builder seems to auto find correct device name).